CMakeLists.txt

```diff
@@ -69,10 +58,6 @@ install(TARGETS pass_through_filter_point_cloud2
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
-# Install headers
```

Comment:
As it is not expected any other package would link to this one, installing the includes does not make sense. It would only tempt users to write the includes, but then they'd find out they have no way to link to this plugin.